spec/roundrect.md

```diff
@@ -17,17 +17,39 @@ Proposal
 --------
 
 ```webidl
+
+
 interface mixin CanvasPath {
   // all doubles are unrestricted.
-  void roundRect(double x, double y, double w, double h, sequence<double> radius);
+  void roundRect(unrestricted double x, unrestricted double y, unrestricted double w, unrestricted double h, sequence<(unrestricted double or DOMPoint)> radii);
 };
 ```
 
 `radius` specifies the radius of corners. Each corner is represented by a single radius.
 
-If `radius.length == 1` then all 4 corners have the same length.
+If `radii.length == 1` then all 4 corners have the same length.
+
+If `radii.length == 2` then the first value applies to the top-left and bottom-right corners, and the second value applies to the top-right and bottom-left corners.
+
+If `radii.length == 3` then the first value applies to the top-left corner, the second value applies to the top-right and bottom-left corners, and the third value applies to the bottom-right corner.
+
+If `radii.length == 4` then each corner is specified, in order: top-left, top-right, bottom-right, bottom-left.
+
+If `w` and `h` are both greater than  or equal to 0, or if both are smaller than 0, then the primitive is drawn clockwise. Otherwise, it is drawn conterclockwise.
```

Comment:
*Typo:* double space in "greater.than..or".

Also this [counter]clockwise thing sounds a bit open to interpretation and confusing. Wouldn't reusing a similar prose to the one for [rect()](https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-rect) be clearer? (defining which points are to be drawn in which order no matter the input's values).  
I'm even wondering if I'm not missing the point and instead of just saying it should behave like current `rect()` it also says the `radii` order should be determined by this?